OtherVersions.md

```diff
@@ -90,4 +90,4 @@ The following 3 tasks can be used for deployment:
 
 Deploying any of these to the roboRIO will disable the current startup project until it is redeployed.
 
-From here, ssh into the roboRIO using the `admin` account (`lvuser` will fail to run in many cases). In the admin home directory, a file for each deploy type will exist (`myRobotCpp`, `myRobotCppStatic` and `myRobotJavaRun`). These can be run to start up the corresponding project.
+From here, ssh into the roboRIO using the `lvuser` account and run `frcRunRobot.sh` (It's in path).
```

Comment:
frcRunRobot.sh is run automatically by lvrt.

Reply:
It is, but when you deploy myRobot it stops the robot code. And then when you run frcRunRobot.sh from the terminal, it runs robot command manually, and pipes the output to the terminal.